lib/services/rally.rb

```diff
@@ -41,6 +41,11 @@ def dont_send_releases?
     data.dont_send_releases == "1"
   end
 
+  def active_workspace_configuration
+    workspace = meta_data.workspaces.detect {|w| w.ObjectID == data.workspace_id } rescue {}
```

Comment:
There's enough going on here that blanket rescue feels a bit risky in the future.  If there's ever a typo on this line, or one of methods being called changes, it won't be obvious. What are we specifically trying to avoid here?

For the line below, if we're just trying to avoid `[] undefined` because configuration may not be there, we could use [`Hash#dig`](http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig) eg:

```ruby
workspace.dig("Configuration", "WorkspaceConfiguration")
```